pandas-stubs/core/series.pyi

```diff
@@ -726,15 +726,27 @@ class Series(IndexOpsMixin[S1], NDFrame):
     def unique(self) -> np.ndarray: ...
     @overload
     def drop_duplicates(
-        self, *, keep: NaPosition | Literal[False] = ..., inplace: Literal[False] = ...
+        self,
+        *,
+        keep: NaPosition | Literal[False] = ...,
+        inplace: Literal[False] = ...,
+        ignore_index: _bool = ...,
     ) -> Series[S1]: ...
     @overload
     def drop_duplicates(
-        self, *, keep: NaPosition | Literal[False] = ..., inplace: Literal[True]
+        self,
+        *,
+        keep: NaPosition | Literal[False] = ...,
+        inplace: Literal[True],
+        ignore_index: _bool = ...,
     ) -> None: ...
```

Comment:
Same comment as within `frame.pyi` - reverse the 2 overloads (looks like they were not using the consistent pattern before)